### ISSUE TITLE
Learn from Balloon Chain people

### ISSUE THREAD

**Issue by sandover** [sandover/kraken#51]
Linden to help arrange

**Comment by sandover** [comment]
@coderodeoclown I have passed you the ball here, OK?  

cc @tiggrum

**Comment by tiggrum** [comment]
FYI, I texted Evelyn earlier today. She said she is available for calls during the day M-F

**Comment by tiggrum** [comment]
I’ll let Linden take it from here.

**Comment by coderodeoclown** [comment]
Yup, I messaged her too. Brandon, what's your general avail for a
conference call? And who else do you want on the call so I can coord avails?

**Comment by sandover** [comment]
My wednesday and thursday PM after 3 both look good. Can make anything work Saturday.

If Liz Vowell ((512) 779-2627) is also available that might be good.

Brandon

**Comment by sandover** [comment]
This was a good call.  @coderodeoclown I think there might been a next step or two.  Could you make fresh issues here, if so.  

Either way, once that's done (or not done), please close this one.
